tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/pixel_size.tsx

```diff
@@ -9,5 +9,7 @@ export const pixel_size: Feature<number> = {
     1.5: 'Pixel Perfect 1.5x',
     2: 'Pixel Perfect 2x',
     3: 'Pixel Perfect 3x',
+    4: 'Pixel Perfect 4x',
+    4.5: 'Pixel Perfect 4.5x',
```

Comment:
there is no reason this couldn't be an arbitrary input or a slider for the record